config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -30,8 +30,8 @@ periodics:
         -project:kubernetes/169
         NOT "complete the pre-review checklist and request an API review"
       - --updated=5m
-      - --token=/etc/github-token/token
-      - --endpoint=http://ghproxy.default.svc.cluster.local
+      - --github-token=/etc/github-token/token
+      - --github-endpoint=http://ghproxy.default.svc.cluster.local
```

Comment:
this isn't a sufficiently safe way to make this change, as we're not running from source here, see the `image` above